### PR TITLE
Introduce docker container for project.

### PR DESCRIPTION
**Pull request by onibox**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,16 @@ For an easy and fast deployment of your application we recommend [heroku](https:
...
 <a href="https://heroku.com/deploy?template=https://github.com/sphereio/sphere-sunrise"><img src="https://www.herokucdn.com/deploy/button.png" alt="Deploy"></a>
 
+## Docker
+
+[![Docker build](http://dockeri.co/image/sphereio/sunrise)](https://registry.hub.docker.com/u/sphereio/sunrise/)
+
+You can also use a ready-to-run docker container.
+
+```bash
+$ docker run -it --rm -p 9000:9000 sphereio/sunrise
```
Comment (svenmueller) [inline README.md]: how do i pass credentials etc? which environment variables are supported?

Comment (onibox) [inline README.md]: If you don't provide them, you will be presented with a formular requiring you to enter them.

Comment (svenmueller) [inline README.md]: but how do i provide it? is it possible via environment variables?

Comment (onibox) [inline README.md]: Yes, have a look at the README regarding integration tests. There you see the following list of environment variables:

```bash
export SPHERE_SUNRISE_IT_PROJECT="your-project-key"
export SPHERE_SUNRISE_IT_CLIENT_SECRET="your-the-client-secret"
export SPHERE_SUNRISE_IT_CLIENT_ID="your-client-id"
```
which you can provide via the `-e` docker parameter.

Comment (svenmueller) [inline README.md]: cool...thx a lot! :+1:

**Pull request merged**